submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/notifications/templates/SuccessfulSubmissionTemplate.kt

```diff
@@ -0,0 +1,58 @@
+package ac.uk.ebi.biostd.notifications.templates
+
+import ebi.ac.uk.base.isNotBlank
+import ebi.ac.uk.notifications.integration.model.NotificationTemplate
+import ebi.ac.uk.notifications.integration.model.NotificationTemplateModel
+
+internal class SuccessfulSubmissionTemplate : NotificationTemplate<SuccessfulSubmissionModel>("") {
+    override fun getContent(model: SuccessfulSubmissionModel): String = """
```

Comment:
not really a fan of template content in code, why we just not use the same approach as plain resource files (as with HTML email), it is are simpler/easier to control and edit.